Portofolio Page v1 Mobile Layout Implementation

math-marcellino opened the pull request:
## 1. Implement Mobile Layout for PortofolioPage
![image](https://user-images.githubusercontent.com/81855912/159851357-9b30db11-46d2-423e-9d9f-2bfcac6746f8.png)
![image](https://user-images.githubusercontent.com/81855912/159851407-5f5f5b32-6035-4494-8d28-ac9fbc7a18c5.png)

## 2. Fix Minor Styling
- Fix max and min column size for two column layout
- Implement minor styling from figma

Note:
- The `Return` value is still hard coded

math-marcellino committed:
Remove unused imports

math-marcellino committed:
Fix Two Column Layout Sizing

math-marcellino committed:
Revert "Fix Two Column Layout Sizing"

This reverts commit 4b2988f85245de2b4d4c5cab8cc9fe4476b5c4e5.

math-marcellino committed:
Fix Two Column Layout Sizing

math-marcellino committed:
Implement Mobile Layout

math-marcellino committed:
Fix minor styling

pyk commented on components/v1/PortofolioPage/PortofolioPage.tsx:
```diff
@@ -122,16 +124,26 @@ const PortofolioPage: FunctionComponent<PortofolioPageProps> = ({}) => {
+                                                        <div className="flex flex-col space-y-[8px]">
+                                                            <p className="text-gray-light-12 dark:text-gray-dark-12">{metadata.title}</p>
+                                                            <p className="text-gray-light-10 dark:text-gray-dark-10 lg:hidden">
+                                                                {formattedEthriseBalance.toFixed(2)} {metadata.title}
```
@math-marcellino Please use `tokenBalanceFormmater` for this

math-marcellino replied:
Okay sir, but if we use `tokenBalanceFormatter`, there will be more than 2 decimal behind the coma, which is different from the design provided in figma, is that okay?
![image](https://user-images.githubusercontent.com/81855912/159872525-f5aff6ce-eed8-42af-b5b8-ceaa9d0ba7ec.png)

pyk replied:
@math-marcellino it's okay. Coz we want to make display amount consistent across the app. May be we want to remove the symbol on the `Amount` column? 

cc @ayikfour

math-marcellino replied:
Okay noted sir @pyk . Btw I also agree to remove the symbol from `Amount` column, because it is too long and it is already displayed on the `Token` column.

cc @ayikfour

pyk commented on components/v1/PortofolioPage/PortofolioPage.tsx:
```diff
@@ -122,16 +124,26 @@ const PortofolioPage: FunctionComponent<PortofolioPageProps> = ({}) => {
                                                     </td>
-                                                    <td className="text-gray-light-10 dark:text-gray-dark-10">
+                                                    <td className="hidden text-gray-light-10 dark:text-gray-dark-10 lg:table-cell">
                                                         {formattedEthriseBalance.toFixed(2)} {metadata.title}
```
@math-marcellino Please use `tokenBalanceFormmater` for this

pyk commented on components/v1/PortofolioPage/PortofolioPage.tsx:
```diff
@@ -159,13 +171,23 @@ const PortofolioPage: FunctionComponent<PortofolioPageProps> = ({}) => {
+                                                        <div className="flex flex-col space-y-[8px]">
+                                                            <p className="text-gray-light-12 dark:text-gray-dark-12">{metadata.vaultTitle}</p>
+                                                            <p className="text-gray-light-10 dark:text-gray-dark-10 lg:hidden">
+                                                                {formattedRvETHUSDCBalance.toFixed(2)} {metadata.vaultTitle}
```
@math-marcellino Please use `tokenBalanceFormmater` for this

pyk commented on components/v1/PortofolioPage/PortofolioPage.tsx:
```diff
@@ -159,13 +171,23 @@ const PortofolioPage: FunctionComponent<PortofolioPageProps> = ({}) => {
+                                                        </div>
+                                                    </td>
+                                                    <td className="hidden text-gray-light-10 dark:text-gray-dark-10 lg:table-cell">
+                                                        {formattedRvETHUSDCBalance.toFixed(2)} {metadata.vaultTitle}
```
@math-marcellino Please use `tokenBalanceFormmater` for this (without toFixed)

pyk commented on components/v1/PortofolioPage/PortofolioPage.tsx:
```diff
@@ -159,13 +171,23 @@ const PortofolioPage: FunctionComponent<PortofolioPageProps> = ({}) => {
+                                                    <td className="hidden text-green-light-11 dark:text-green-dark-11 lg:table-cell">+$13.34</td>
+                                                    <td>
+                                                        <div className="flex flex-col space-y-[8px]">
+                                                            <p className="text-gray-light-12 dark:text-gray-dark-12">{dollarFormatter.format(latestVaultExchangeRateFormatted * formattedRvETHUSDCBalance)}</p>
```
@math-marcellino Please use raw value instead of fomatted one (`exchangeRate * rvETHUSDCBalance`)

pyk reviewed (changes requested):
@math-marcellino Made few changes proposal, please take a look. Ty!

math-marcellino committed:
Fix Variable & Balance Format

- Change some variable name (to avoid misunderstanding)
- use `tokenBalanceFormatter` to format asset balance
